AppDB/appscale/datastore/groomer.py

```diff
@@ -556,6 +557,15 @@ def clean_up_indexes(self, direction):
       start_key = ''
     end_key = dbconstants.TERMINATING_STRING
 
+    # Indicate that an index scrub has started after the journal was removed.
+    if direction == datastore_pb.Query_Order.ASCENDING and not start_key:
+      index_state = self.db_access.get_metadata(
+        cassandra_interface.INDEX_STATE_KEY)
+      if index_state is None:
+        self.db_access.set_metadata(
+          cassandra_interface.INDEX_STATE_KEY,
+          cassandra_interface.IndexStates.SCRUB_IN_PROGRESS)
```

Comment:
There's no behavioral difference for the deployment between an unset key and `SCRUB_IN_PROGRESS`, so staying in that state for awhile shouldn't be a problem.

Once the groomer run again and gets to the end of scrubbing the kind index, this index will be marked as clean.